src/components/IterableNestedComponent.tsx

```diff
@@ -0,0 +1,134 @@
+import autobind from "autobind-decorator"
+import * as React from "react"
+import * as _ from "lodash"
+import { NestedEditComponent, InlineComponentProps } from "../CruxComponent"
+
+interface IterableNestedComponentProps extends InlineComponentProps {
+    model: any
+    collapsable: boolean
+    totalLength: number
+    collapseNestedToggle: Function
+    getIterableNestedTitle: Function
+    remove: Function
+    addAtIndex: Function
+    reorder: Function
+}
+@autobind
+export class IterableNestedComponent extends React.Component<IterableNestedComponentProps, any> {
+
```

Comment:
New Component for Iterable Nested. Here Components it will be rendered only when the Nested Iterable Collapse is open.Hence It is improving the performance. And also mouseEnter and mouseLeave action has been handled in this component